frontend/packages/noobaa-storage-plugin/src/components/capacity-breakdown/capacity-breakdown-card.scss

```diff
@@ -1,7 +1,19 @@
+@import '~@patternfly/patternfly/sass-utilities/all';
+
 .nb-capacity-breakdown-card__body {
-    padding-top: 24px;
+  padding-top: 24px;
```

Comment:
`px` -> `rem` or `em`

Reply:
So the linter just spaced this part I will remove this change as I don't wanna touch old CSS.